march_hardware/src/Joint.cpp

```diff
@@ -84,23 +84,23 @@ float Joint::getTemperature()
 
 IMotionCubeState Joint::getIMotionCubeState()
 {
-  // Return an object of strings:
-  // the literal bits of the Status Word, Detailed Error and Motion Error
-  // and the parsed interpretation of these bits
-  IMotionCubeState states;
-
-  std::bitset<16> statusWordBits = this->iMotionCube.getStatusWord();
-  states.statusWord = statusWordBits.to_string();
-  std::bitset<16> detailedErrorBits = this->iMotionCube.getDetailedError();
-  states.detailedError = detailedErrorBits.to_string();
-  std::bitset<16> motionErrorBits = this->iMotionCube.getMotionError();
-  states.motionError = motionErrorBits.to_string();
-
-  states.state = this->iMotionCube.getState(this->iMotionCube.getStatusWord());
-  states.detailedErrorDescription = this->iMotionCube.parseDetailedError(this->iMotionCube.getDetailedError());
-  states.motionErrorDescription = this->iMotionCube.parseMotionError(this->iMotionCube.getMotionError());
-
-  return states;
+    IMotionCubeState states;
+
+    std::bitset<16> statusWordBits = this->iMotionCube.getStatusWord();
+    states.statusWord = statusWordBits.to_string();
+    std::bitset<16> detailedErrorBits = this->iMotionCube.getDetailedError();
+    states.detailedError = detailedErrorBits.to_string();
+    std::bitset<16> motionErrorBits = this->iMotionCube.getMotionError();
+    states.motionError = motionErrorBits.to_string();
+
+    states.state = this->iMotionCube.getState(this->iMotionCube.getStatusWord());
+    states.detailedErrorDescription = this->iMotionCube.parseDetailedError(this->iMotionCube.getDetailedError());
+    states.motionErrorDescription = this->iMotionCube.parseMotionError(this->iMotionCube.getMotionError());
+
+    states.motorCurrent = this->iMotionCube.getMotorCurrent();
+    states.motorVoltage = this->iMotionCube.getMotorVoltage();
+
+    return states;
```

Comment:
What is this?

Reply:
Autoformatting, I guess.

On magic numbers: should they really be IMC constants when they are only used in their specific functions? I thought it would be best to just limit them to the function scope

Reply:
That's fine too, it's up to you if you define them at the top of the method or where needed, but keep in mind the `const` declarator and

![image](https://user-images.githubusercontent.com/9715314/61779596-43f76d80-ae01-11e9-8235-0dd2edb80502.png)